scripts/ci/make/tasks

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+######################################################
+###   PROJECT SPECIFIC FUNCTIONS - RUNTIME ORDER   ###
+######################################################
+
+build() {
+    # dont use the docker build cache if building from a tag
+    print_colorized WARN "building image ${TEST_IMAGE_NAME}"
+    if [[ -n "${GIT_TAG}" ]]; then
+        local build_cache="--no-cache"
```

Comment:
abstractions like these should have some way to advertise themselves either on terminal output or a log file. When debugging why things aren't working exactly right, or trying to extend/modify what is currently being done, the commands should be logged somewhere. Is it possible that these are being logged and I am just not seeing it?

Reply:
added a print statement there